src/file-picker/src/FilePicker.js

```diff
@@ -174,7 +174,9 @@ export default class FilePicker extends PureComponent {
   }
 
   handleBlur = e => {
-    if (e && e.target) e.target.files = this.state.files
+    // Setting e.target.files to an array fails. It must be a FileList
+    if (e && e.target) e.target.files = this.fileInput.files
```

Comment:
We'll need to ensure we check that `this.fileInput` is not null.

Reply:
done